jcl/src/java.base/share/classes/com/ibm/tools/attach/target/CommonDirectory.java

```diff
@@ -269,7 +303,7 @@ public static int countTargetDirectories () {
 			return 0;
 		} else {
 			int count = 0;
-			for (int i=0; i < vmDirs.length; ++i) {
+			for (int i= 0; i < vmDirs.length; ++i) {
```

Comment:
If you want to fix this, which isn't required, there should be a space before the = as well